ui/txs/noves/useDescribeTxs.tsx

```diff
@@ -0,0 +1,89 @@
+import type { UseQueryResult } from '@tanstack/react-query';
+import _ from 'lodash';
+
+import type { NovesDescribeTxsResponse } from 'types/api/noves';
+import type { Transaction } from 'types/api/transaction';
+
+import type { ResourceError } from 'lib/api/resources';
+
+export interface DescribeTxs {
+  txHash: string;
+  type?: string;
+  description?: string;
+  isLoading: boolean;
+  enabled: boolean;
+}
+
+export interface TransactionWithTranslate extends Transaction {
+  translate?: DescribeTxs;
+}
+
+export default function useDescribeTxs(items: Array<Transaction> | undefined, translateEnabled: boolean) {
+  const txsHash = items?.map(i => i.hash);
+  const txsChunk = _.chunk(txsHash || [], 10);
+
+  const txsParams = Array(5).fill(undefined).map((_, i) => txsChunk[i]);
```

Comment:
Shouldn't we pass the chunks directly into the `useFetchTxs` hook? I don't really understand the reason for formatting it in this manner.